tiled/src/main/kotlin/ktx/tiled/tiledMaps.kt

```diff
@@ -153,3 +154,18 @@ inline fun TiledMap.forEachMapObject(layerName: String, action: (MapObject) -> U
     action(it)
   }
 }
+
+/**
+ * Extension method to run a [lambda] on a specific type of [layers][MapLayer] of the [TiledMap]. The lambda
+ * takes the matching [MapLayer] as a parameter.
+ *
+ * The class matching is an exact match meaning that a given subclass like [TiledMapTileLayer] will not match
+ * for the argument [MapLayer].
+ */
+inline fun <reified T : MapLayer> TiledMap.forEachLayer(lambda: (T) -> Unit) {
```

Comment:
Rename lambda to something along the lines of `operation` or `action`. `forEach` in stdlib uses `action`.